daemon/src/model/cfd.rs

```diff
@@ -689,6 +689,10 @@ impl Cfd {
         (self.order.term * Cfd::REFUND_THRESHOLD).as_blocks().ceil() as u32
     }
 
+    pub fn expiry_timestamp(&self) -> i64 {
+        self.order.oracle_event_id.as_unix_timestamp()
```

Comment:
Ideally this is stored as a field on the CFD and we don't have to parse it from the oracle event id but I didn't want to change the data model.